src/lib/core/overlay/overlay-container.spec.ts

```diff
@@ -0,0 +1,55 @@
+import {async, inject, TestBed} from '@angular/core/testing';
+import {Component, NgModule, ViewChild, ViewContainerRef} from '@angular/core';
+import {PortalModule, TemplatePortalDirective} from '../portal/portal-directives';
+import {Overlay, OverlayContainer, OverlayModule} from './index';
+
+describe('OverlayContainer', () => {
+  let overlay: Overlay;
+  let overlayContainer: OverlayContainer;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [OverlayTestModule]
+    }).compileComponents();
+  }));
+
+  beforeEach(inject([Overlay, OverlayContainer], (o: Overlay, oc: OverlayContainer) => {
+    overlay = o;
+    overlayContainer = oc;
+  }));
+
+  it('should remove the overlay container element from the DOM on destruction', () => {
+    const fixture = TestBed.createComponent(TestComponentWithTemplatePortals);
+
+    const overlayRef = overlay.create();
+    overlayRef.attach(fixture.componentInstance.templatePortal);
+    fixture.detectChanges();
+
+    expect(document.body.querySelector('.cdk-overlay-container'))
```

Comment:
Does this pass? I know that when I did the cleanup last time, there were still some divs left over between tests because the overlay container didn't have any cleanup logic.